packages/medusa/src/api/routes/store/products/index.ts

```diff
@@ -43,3 +45,7 @@ export type StorePostSearchRes = {
 export type StoreProductsListRes = PaginatedResponse & {
   products: Product[]
 }
+
+export type StoreProductsListTypesRes = {
+  types: ProductType[]
```

Comment:
todo: If the endpoint is not removed, I think we should change the response to `product_types` to eliminate ambiguity around what types, we are working with

Reply:
Done, see 0ee84569846c46b5fa54227249dbbafd10b87715